playbooks/hokusai.md

```diff
@@ -43,7 +43,9 @@ Prerequisite: install the AWS IAM Authenticator plugin
 brew install aws-iam-authenticator
 ```
 
-_Make sure IAM credentials are set in AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY or ~/.aws/credentials!_
+_Make sure IAM credentials are set in your shell via AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY environment variables or ~/.aws/credentials!_
+
+Also set `AWS_DEFAULT_REGION=us-east-1`
```

Comment:
Another option might be to have engineers run `aws configure`, which will prompt for credentials and configuration options to populate `~/.aws/credentials` and `~/.aws/config` reliably.

This is also safe to run multiple times:

```
$ aws configure
AWS Access Key ID [None]: ****************VFET
AWS Secret Access Key [None]: ****************OmV3
Default region name [None]: us-east-1
Default output format [None]: json

$ aws configure
AWS Access Key ID [****************VFET]: 
AWS Secret Access Key [****************OmV3]: 
Default region name [eu-west-2]: us-east-1
Default output format [json]:
```

Reply:
👍

Reply:
Good to have both options.